src/pages/views.py

```diff
@@ -40,3 +40,16 @@ def about_view(request):
 
 def contact_view(request):
     return render(request, 'pages/contact.html', {})
+
+
+def notify_view(request):
+    # send email and stuff

```

Comment:
Mark with
```python
# TODO: 
```
So it can be easily found with __PyCharm__

Reply:
ok